website/docs/examples/examples_index.md

```diff
@@ -5,6 +5,18 @@ title: Examples Index
 
 # Amiga SDK Examples
 
+:::caution Version Compatibility
+The examples in this documentation may change with each release of the farm-ng-amiga software.
+Some examples might use methods that are not yet released.
+
+- **To run the latest examples**, you should build the software from the [source](/docs/brain/brain-install#install-from-source-advanced).
+Clone the repository and run `pip install .` within the cloned directory.
+- **To use examples compatible with the stable release**, simply install the released version using
+`pip install farm-ng-amiga`.
+
```

Comment:
@guilhermedemouraa I think missing is that they should check out the version of the examples corresponding to that release.

E.g.,
```
cd farm-ng-amiga/
git checkout <v2.x.y>
cd py/examples/camera_client/
```

Reply:
Good catch. I will follow up on a new ticket (and make sure it's not set to auto-merge 😅